Main.sublime-menu

```diff
@@ -1,9 +1,74 @@
 [
-	{
-        "id": "goto",
-        "children":
-        [
-            { "command": "typescript_nav_to", "caption": "Navigate to TypeScript Symbol..."}
+  {
+    "id": "goto",
+    "children": [
+      {
+        "command": "typescript_nav_to",
+        "caption": "Navigate to TypeScript Symbol..."
+      }
+    ]
+  },
+  {
+    "id": "preferences",
+    "children": [
+      {
+        "id": "package-settings",
+        "children": [
+          {
+            "caption": "TypeScript",
+            "children": [
+              {
+                "command": "open_file",
+                "args": {
+                  "file": "${packages}/TypeScript/Preferences.sublime-settings"
```

Comment:
You are right that is a problem in that case, though there seems to be no way around it.